lib/project-data-updater.js

```diff
@@ -88,8 +86,7 @@ ProjectDataUpdater.prototype.checkForAndImportUpdates = function(info) {
         .then(done, done);
     });
   } else {
-    this.done(new Error(this.fullName + ': no ' +
-      module.exports.ABOUT_YML + ' updates imported'));
+    this.done(true);
```

Comment:
Actually, you can call `this.done()`, and the tests can check for `expect(err).to.be.undefined`, which is (as far as I can tell in my limited Node experience) the common Node idiom.